qa/L0_FW_iterators/test.sh

```diff
@@ -12,6 +12,8 @@ NUM_GPUS=$(nvidia-smi -L | wc -l)
 
 test_body() {
     python test_RN50_data_fw_iterators.py --gpus ${NUM_GPUS} -b 13 --workers 3 --prefetch 2 -i 100 --epochs 2
+
+    nosetests --verbose test_fw_iterators_detection.py
```

Comment:
For this to work you need to add `source qa/setup_dali_extra.sh` somewhere above to setup DALI extra

Reply:
Done